tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #61.            # To make sure the nonlinear iteration index does not interfere with the 
</pre><p> </p><p>I've run the tutorial on my local machine and it works, but I find it quite hard to follow the exporting logic when time and iterations are combined.</p><p> </p><p>If I understand correctly, the output files, say, for 1d domains will be: </p><p> </p><p>debug_iterations_1_000000.pvd  # time = 0 sec</p><p>debug_iterations_1_000101.pvd  # time = 1 sec; iteration = 1</p><p>debug_iterations_1_000102.pvd  # time = 1 sec; iteration = 2</p><p>debug_iterations_1_000201.pvd  # time = 2 sec; iteration = 1</p><p> </p><p>Is that correct? I was wondering if ParaView will be pleased if we add yet another "_" . Something like:</p><p> </p><p>debug_iterations_1_000000_1.pvd  # time = 0 sec</p><p>debug_iterations_1_000001_1.pvd  # time = 1 sec; iteration = 1</p><p>debug_iterations_1_000001_2.pvd  # time = 1 sec; iteration = 2</p><p>debug_iterations_1_000002_1.pvd  # time = 2 sec; iteration = 1</p><p> </p><p>Have you tried something like this?</p><p> </p><p>In any case, it might be beneficial to add a markdown cell below each mixin model with the outputs that it will generate (not all outputs, perhaps just for the bulk or the fracture), with some small in-line comment as included above. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>In agreement with @Jhabriel. It is necessary to explain increasing indices with more elaborate suffixes.</p>